tests/test_classifier/test_class_prediction_error.py

```diff
@@ -65,7 +65,7 @@ def test_class_prediction_error_quickmethod(self):
         ax = fig.add_subplot()
 
         clf = LinearSVC(random_state=42)
-        g = class_prediction_error(clf, X, y, ax)
+        g = class_prediction_error(clf, X, y, ax, random_state=42)
```

Comment:
Gave the quick method a random state argument and removed the random state hard coding in that function.